pages/locations.md

```diff
@@ -0,0 +1,76 @@
+---
+description: Locations
```

Comment:
This gets used for the page's meta description, so perhaps put something more descriptive here. Perhaps:

> List of Buildit's global studios with addresses and directions.

Reply:
Yea, that one is better.